pipeline/steps/start-stop-vm.yaml

```diff
@@ -0,0 +1,67 @@
+parameters:
+  - name: env
+    type: string
+  - name: pipelineStart
+    type: boolean
+  - name: subscriptionName
+    type: string
+
+steps:
+  - task: AzureCLI@2
+    displayName: Start and Stop Vms during pipeline run
+    name: startStopVm
+    condition: always()
+    inputs:
+      azureSubscription: '${{ parameters.subscriptionName }}'
+      scriptType: pscore
+      scriptLocation: inlineScript
+      inlineScript: |
+        Write-Output "Task started: startStopVm"
+        $turn_off_vm_post_run="$env:TURN_OFF_VM_POST_RUN"
+        $resourcegroup="cvp-recordings-${{parameters.env}}-rg"
+        $pipelineStart="${{parameters.pipelineStart}}"
+
+        $rgExists = az group exists --name $resourcegroup
+        if ( $rgExists )
+        {
+          Write-Output "$resourcegroup exists in subscription"
+          $vmList = az vm list --resource-group $resourcegroup  --query "[].{name:name}"  | ConvertFrom-Json
+          $vmListCnt=$vmList.Count 
+          Write-Output "$vmListCnt found in resource group"
```

Comment:
guess this file was used for testing so can be removed now?

Reply:
yeah, template was originally here but was moved to azure-devops-templates repo. Has been removed